upgrade-instructions.md

```diff
@@ -15,9 +15,33 @@ Note: this section is in progress. It will be updated during all the development
 Changes to apply between 3.6.x and 3.7.x will be reported here.
 
 Change list:
-
+ - [Adding the threadId to the ElasticSearch index](#adding-the-threadid-to-the-elasticsearch-index)
  - [Rework message denormalization](#rework-message-denormalization)
```

Comment:
While we are at it `### Adding threadId column to message metadata tables` is missing from this menu...

Can you add it?